pyfar/signals/files.py

```diff
@@ -348,11 +348,12 @@ def head_related_impulse_responses(
             degrees.
         ``'median'``
             Return median plane HRIRs with an angular resolution of 2 degrees.
-        List of coordinates
-            Return HRIRs at specific positions defined by a list of azimuth
-            and elevation values in degrees. For example
+        Array like
+            Return HRIRs at azimuth and elevation in degrees. For example
             ``[[30, 0], [330, 0]]`` returns HRIRs on the horizontal plane
             (0 degree elevation) for azimuth angles of 30 and 330 degrees.
+        :py:class:`~pyfar.Coordinates`
+            Return HRIRs at positions defined by a pyfar Coordinates object.
```

Comment:
I'd suggest adding the info, that all radii are set to 1.7 m here, or that the radii are ignored.

Reply:
good point - I've added a note on that.

Reply:
Is it clear how the coorinates are defined? or do we need to metion sth like the view is at (1, 0, 0) and up at (0,0,1) or that we suggest to define them in spherical_elevation coordinates?